lib/src/get_position.dart

```diff
@@ -53,29 +60,29 @@ class GetPosition {
     final box = key!.currentContext!.findRenderObject() as RenderBox;
     final bottomRight =
         box.size.bottomRight(box.localToGlobal(const Offset(0.0, 0.0)));
-    return bottomRight.dy;
+    return bottomRight.dy + padding!.bottom;
   }
 
   ///Get the top position of the widget
   double getTop() {
     final box = key!.currentContext!.findRenderObject() as RenderBox;
     final topLeft = box.size.topLeft(box.localToGlobal(const Offset(0.0, 0.0)));
-    return topLeft.dy;
+    return topLeft.dy - padding!.top;
```

Comment:
Are you sure that the padding won't be null here?

Reply:
I have updated the PR. please review the changes.